crates/fluvio/src/lib.rs

```diff
@@ -10,7 +10,11 @@
 //!
 //! # Prerequisites
 //!
-//! [Install Fluvio](#installation)
+//! Add `fluvio` crate to the project's `Cargo.toml` file
+//!
+//! ```ignore
+//! fluvio = "0.12"
+//! ```
```

Comment:
Give that this could be trivial, I was considering the fact of removing it at all.
Perhaps a link to the examples page on https://www.fluvio.io/api/official/rust/examples/ could be helpful to invite crate consumers reading this documentation from an IDE for instance.